website/docusaurus.config.js

```diff
@@ -59,16 +77,29 @@ module.exports = {
       ],
       logo: {
         alt: 'Facebook Open Source Logo',
-        src: 'https://docusaurus.io/img/oss_logo.png',
+        src: 'img/oss_logo.png',
         href: 'https://opensource.facebook.com',
       },
       copyright: `Copyright © ${new Date().getFullYear()} Facebook, Inc. Built with Docusaurus.`,
     },
+    prism: {
+      defaultLanguage: 'shell',
+      theme: require('prism-react-renderer/themes/github'),
+      darkTheme: require('prism-react-renderer/themes/dracula'),
```

Comment:
I'm wondering are these two themes used by React Native and React website? If not, can we use the same ones?

Reply:
On the React Native website we use custom prism theme, which always render code block on dark background, no matter of theme selected by user. If it's not an issue for you, I can import the theme in.